aspnet/fundamentals/configuration.rst

```diff
@@ -49,11 +48,11 @@ The settings required by your application and the mechanism used to specify thos
 .. note:: You could store your ``Configuration`` instance as a service, but this would unnecessarily couple your application to a single configuration system and specific configuration keys. Instead, you can use the :ref:`Options pattern <options-config-objects>` to avoid these issues.
 
 Using the built-in sources
-----------------------------
+--------------------------
 
-The configuration framework has built-in support for JSON, XML, and INI configuration files, as well as support for in-memory configuration (directly setting values in code) and the ability to pull configuration from environment variables and command line parameters. Developers are not limited to using a single configuration provider. In fact several may be set up together such that a default configuration is overridden by settings from another provider if they are present.
+The configuration framework has built-in support for JSON, XML, and INI configuration files, as well as support for in-memory configuration (directly setting values in code) and the ability to pull configuration from environment variables and command line parameters. Developers are not limited to using a single configuration source. In fact several may be set up together such that a default configuration is overridden by settings from another source if they are present.
```

Comment:
> as well as support for in-memory configuration (directly setting values in code) 

This reminds me of something: Although it is up there in the first example I think I haven't seen any mention in this article about the fact that configuration is mutable, e.g. you can do `config["somekey"] = "somevalue"`. There is no persistence for this but it is supported with all providers (or at least it should) and values set like this are kept in memory in the same instance until you call `Reload()`.